### PR TITLE
config: cleanup

### PR DESCRIPTION
**Pull request by dajohi**
Remove default values from descriptions as some are wrong.  Also, check
votebitsextended early.

### CODE REVIEW

**Comment by raedah** [comment]
I suppose if we just remove the default values, then we dont have to worry about maintaining the correct values, but how are users suppose to know what the default value is? If the description used variables, the default values could be set once and automatically dropped in.

**Comment by jrick** [comment]
--help will display the values that would be used if otherwise unset with a cli flag.  It starts off with the defaults, parses the config file, and then displays the help based on that.

That is better than printing the actual defaults in the message since those values are rather misleading when they have been changed in a config file.  It will use the config file's settings over the defaults.

File: config.go [hunk 1]
```diff
@@ -684,17 +688,24 @@ func loadConfig() (*config, []string, error) {
...
+		fmt.Fprintln(os.Stderr, err.Error())
+		fmt.Fprintln(os.Stderr, usageMessage)
+		return loadConfigError(err)
 	}
-
-	// Set ticketfee and txfee to defaults if none are set.  Avoiding using default
-	// confs because they are dcrutil.Amounts
-	if cfg.TicketFee == 0.0 {
-		cfg.TicketFee = wallet.DefaultTicketFeeIncrement.ToCoin()
+	if len(voteBitsExtendedB) > stake.MaxSingleBytePushLength-2 {
```
Comment (jrick) [inline config.go]: still unsure about this. I think we need to enforce that it is some exact length

Comment (dajohi) [inline config.go]: made it require len of 8 for now.

File: config.go [hunk 2]
```diff
@@ -684,17 +688,31 @@ func loadConfig() (*config, []string, error) {
...
+	voteBitsExtendedB, err := hex.DecodeString(cfg.VoteBitsExtended)
+	if err != nil {
+		err = fmt.Errorf("invalid votebitsextended setting: %v", err)
+		fmt.Fprintln(os.Stderr, err.Error())
+		fmt.Fprintln(os.Stderr, usageMessage)
+		return loadConfigError(err)
 	}
-	if cfg.RelayFee == 0.0 {
-		cfg.RelayFee = txrules.DefaultRelayFeePerKb.ToCoin()
+	if len(voteBitsExtendedB) > stake.MaxSingleBytePushLength-2 {
```
Comment (jrick) [inline config.go]: this is impossible given the above check

Comment (dajohi) [inline config.go]: fixed.

**Pull request merged**